src/README.md

```diff
@@ -0,0 +1,32 @@
+## Extract Method
+
+To automatically build unvalidated language files, perform the following steps:
+
+* Ensure all validated languages have been included in the `src/validated-languages.json` file.
+  * This JSON file must only include the ISO 639-1 language tag (i.e. the language folder name).
+  * Any IETF language tags that do not match a folder name (e.g. `de-CH` or `fr-CA`) will be ignored.
```

Comment:
I'm confused. Can I specify both, ISO 639-1 and IETF now?

- If so, why are you saying it `must only` include ISO-639-1 if it can also be IETF?
- If not, why are you mentioning IETF at all?

Reply:
[IETF languge tags](https://en.wikipedia.org/wiki/IETF_language_tag#Syntax_of_language_tags) actually use ISO 639 as its base.

> A single primary language subtag based on a two-letter language code from ISO 639-1 (2002) or a three-letter code from ISO 639-2 (1998), ISO 639-3 (2007) or ISO 639-5 (2008), or registered through the BCP 47 process and composed of five to eight letters

That's why the root language names are only two letters, except for `fil` (Filipino) which is the exception as it was added in [ISO 639-2](https://en.wikipedia.org/wiki/Filipino_language). I know it's kind of a hodgepodge of specs, but I guess we could reword the spec and say we only want to have folders for the root language (e.g. no `de-CH` folder).

Reply:
I'm out, I don't understand anything regarding this. 🗡

Reply:
me too! 
![](https://user-images.githubusercontent.com/136959/30521490-f49693e0-9b85-11e7-9e08-d75139a38912.gif)